src/components/cluster/detail/certificate_orgs_label.test.js

```diff
@@ -54,17 +54,3 @@ it('renders multiple labels in alphabetic order', () => {
       .props().children
   ).toEqual('Zepto');
 });
-
-it('gives system:masters a special isadmin class', () => {
```

Comment:
Why did you remove this test? as far as I can see the isadmin code is still present.

Reply:
I forgot to remove it. Done now. The extra `isadmin` class makes no sense in combination with the hash colors. At least not with background color.